City-of-Helsinki/drupal-asuntotuotanto-public
# Asu x apply form redirects to application form.

Make fresh, Do not login after finished.

Try to enter any application page, you should be redirected to login page.

Test these cases:

1. Login as admin (Or any other role other than "customer")

-  You should not be able to access any application forms

2. Create new user or login as existing user with role "customer"
  
- Try to access Application page without adding project id.
  
- Try changing the url (for example from "application/add/haso/20" TO application/add/hitas/20)
  - You should automatically create the right form.
  
- Try to access application which application period has expired
